bob-apps/bin/bobd.rs

```diff
@@ -82,38 +87,97 @@ async fn main() {
     }
     warn!("Start listening on: {:?}", addr);
 
-    let (metrics, shared_metrics) = init_counters(&node, &addr.to_string()).await;
-
-    let handle = Handle::current();
-    let bob = BobServer::new(Grinder::new(mapper, &node).await, handle, shared_metrics);
-
-    info!("Start backend");
-    bob.run_backend().await.unwrap();
     info!("Start API server");
     let http_api_port = matches
         .value_of("http_api_port")
         .and_then(|v| v.parse().ok())
-        .unwrap_or(node.http_api_port());
+        .unwrap_or_else(|| node.http_api_port());
     let http_api_address = matches
         .value_of("http_api_address")
         .and_then(|v| v.parse().ok())
-        .unwrap_or(node.http_api_address());
-    bob.run_api_server(http_api_address, http_api_port);
+        .unwrap_or_else(|| node.http_api_address());
 
-    create_signal_handlers(&bob).unwrap();
+    let authentication_type = matches.value_of("authentication_type").unwrap();
```

Comment:
As I see, the authentication type read from command line parameters. This is not the best way to configure authentication. We need to configure it from `node.yaml`